lib/commandLine.ts

```diff
@@ -0,0 +1,60 @@
+import * as yargs from "yargs";
+
+export interface GetArgumentOptions {
+  checkEnvironmentVariables?: boolean;
+  environmentVariableName?: string;
+  defaultValue?: string;
+}
+
+export function getArgument(argumentName: string, options?: GetArgumentOptions): string | undefined {
+  options = options || {};
+
+  let result: string | undefined;
+
+  let rawArgument: unknown = yargs.argv[argumentName];
```

Comment:
Do we need yargs if we're parsing parameters manually anyway?

Reply:
That's a good point. Let me see if I can implement this without using yargs.

Reply:
Nope. `yargs` is still really useful for parsing the command line arguments. `process.argv` just provides the command line arguments as a `string[]`, whereas `yargs` turns them into a map/dictionary. I'll stick with that for now.